### PR TITLE
reformat jobmanager.java file, improve code readability and some other trick refaction.

### PR DESCRIPTION
**Pull request by leechor**
reformat jobmanager.java file, improve code readability, and do some other trick refactoring.
 jobmanager.java execute SQL function too long, split it,

### CODE REVIEW

**Comment by aiwenmo** [comment]
Hi. It's a good PR. Please resolve the code conflict. thx.

File: dlink-app/dlink-app-base/src/main/java/com/dlink/app/flinksql/Submiter.java [hunk 1]
```diff
@@ -123,7 +123,7 @@ public static void submit(Integer id, DBConfig dbConfig) {
         List<StatementParam> trans = new ArrayList<>();
         List<StatementParam> execute = new ArrayList<>();
         for (String item : statements) {
-            String statement = FlinkInterceptor.pretreatStatement(executor, item);
+            String statement = FlinkInterceptor.preTreatStatement(executor, item);
```
Comment (aiwenmo) [inline dlink-app/dlink-app-base/src/main/java/com/dlink/app/flinksql/Submiter.java]: Pretreat statement, not pre treat statement.

**Review by aiwenmo — APPROVED** [review]
You can submit the file optimization first, such as Asserts.java, SqlType.java, Operations.java, GatewayType.java.

**Comment by aiwenmo** [comment]
I don't think it's necessary to change pretreatStatement into preTreatStatement.

**Comment by leechor** [comment]
> 

my IDEA dictionary don't have this word, sorry, I change back to it

**Comment by aiwenmo** [comment]
> > 
> 
> my IDEA dictionary don't have this word, sorry, I change back to it

Looking forward to your optimization.

**Comment by leechor** [comment]
> > > 
> > 
> > 
> > my IDEA dictionary don't have this word, sorry, I change back to it
> 
> Looking forward to your optimization.

I have submitted [that](https://github.com/DataLinkDC/dlink/pull/1135) independently,

**Comment by aiwenmo** [comment]
> > > > 
> > > 
> > > 
> > > my IDEA dictionary don't have this word, sorry, I change back to it
> > 
> > 
> > Looking forward to your optimization.
> 
> I have submitted [that](https://github.com/DataLinkDC/dlink/pull/1135) independently,

Sorry, I didn't see your code submission. Please check your commit.

**Comment by leechor** [comment]
> > > > > 
> > > > 
> > > > 
> > > > my IDEA dictionary don't have this word, sorry, I change back to it
> > > 
> > > 
> > > Looking forward to your optimization.
> > 
> > 
> > I have submitted [that](https://github.com/DataLinkDC/dlink/pull/1135) independently,
> 
> Sorry, I didn't see your code submission. Please check your commit.

I don't get your mean, I have opened a new [pull request](https://github.com/DataLinkDC/dlink/pull/1135)

**Comment by aiwenmo** [comment]
> > > > > > 
> > > > > 
> > > > > 
> > > > > my IDEA dictionary don't have this word, sorry, I change back to it
> > > > 
> > > > 
> > > > Looking forward to your optimization.
> > > 
> > > 
> > > I have submitted [that](https://github.com/DataLinkDC/dlink/pull/1135) independently,
> > 
> > 
> > Sorry, I didn't see your code submission. Please check your commit.
> 
> I don't get your mean, I have opened a new [pull request](https://github.com/DataLinkDC/dlink/pull/1135)

- [X] #1135

**Comment by leechor** [comment]
> > > > > > > 
> > > > > > 
> > > > > > 
> > > > > > my IDEA dictionary don't have this word, sorry, I change back to it
> > > > > 
> > > > > 
> > > > > Looking forward to your optimization.
> > > > 
> > > > 
> > > > I have submitted [that](https://github.com/DataLinkDC/dlink/pull/1135) independently,
> > > 
> > > 
> > > Sorry, I didn't see your code submission. Please check your commit.
> > 
> > 
> > I don't get your mean, I have opened a new [pull request](https://github.com/DataLinkDC/dlink/pull/1135)
> 
> * [x]  [refactor: some trick modification #1135](https://github.com/DataLinkDC/dlink/pull/1135)

@aiwenmo doesn't match checkstyle.xml and spotless_dlink_formatter.xml in the style directory. checkstyle still reports error report error after run spotless:apply.

**Comment by aiwenmo** [comment]
> doesn't match checkstyle.xml and spotless_dlink_formatter.xml in the style directory. checkstyle still reports error report error after run spotless:apply.

Try updating to the latest code.

**Review by aiwenmo — CHANGES_REQUESTED** [review]
Thank you very much for your optimization. Because the file named JobManager.java is complex and critical, and requires a lot of unit tests to verify its interface, please do not optimize it first. Later, I will reconstruct it.

**Comment by leechor** [comment]
> Thank you very much for your optimization. Because the file named JobManager.java is complex and critical, and requires a lot of unit tests to verify its interface, please do not optimize it first. Later, I will reconstruct it.

ok, I close it.

**Comment by leechor** [comment]
> Thank you very much for your optimization. Because the file named JobManager.java is complex and critical, and requires a lot of unit tests to verify its interface, please do not optimize it first. Later, I will reconstruct it.

@aiwenmo I left [JobManager.java](https://github.com/leechor/dlink/blob/ldev_jobmanager/dlink-core/src/main/java/com/dlink/job/JobManager.java) for your reference. it merged the new dev.

**Comment by aiwenmo** [comment]
> > Thank you very much for your optimization. Because the file named JobManager.java is complex and critical, and requires a lot of unit tests to verify its interface, please do not optimize it first. Later, I will reconstruct it.
> 
> @aiwenmo I left [JobManager.java](https://github.com/leechor/dlink/blob/ldev_jobmanager/dlink-core/src/main/java/com/dlink/job/JobManager.java) for your reference. it merged the new dev.

thx. The optimization of other files can be submitted first.

**Pull request closed without merging**